proto/v1/database_service.proto

```diff
@@ -394,4 +446,60 @@ message BackupSetting {
 
   // hook_url(https://www.bytebase.com/docs/administration/webhook-integration/database-webhook) is the URL to send a notification when a backup is created.
   string hook_url = 4;
-}
\ No newline at end of file
+}
+
+// OperationMetadata is the metadata for database operations.
+// Empty for now.
+message OperationMetadata {}
+
+// The message of the backup.
+message Backup {
+  // The type of the backup.
+  enum BackupType {
+    // The type of the backup is unknown.
+    BACKUP_TYPE_UNSPECIFIED = 0;
+
+    // The backup is created by user.
+    MANUAL = 1;
+
+    // The backup is created by automatic backup.
+    AUTOMATIC = 2;
+
+    // The backup is created automatically after doing PITR.
+    PITR = 3;
+  }
+
+  // The state of the backup.
+  enum BackupState {
+    // The state of the backup is unknown.
+    BACKUP_STATE_UNSPECIFIED = 0;
+
+    // The backup is being pending to create.
+    PENDING_CREATE = 1;
+
+    // The backup is ready to use.
+    DONE = 2;
+
+    // The backup is being deleted.
+    FAILED = 3;
+  }
+
+  // The name of the database backup.
```

Comment:
We need to comment on whether the name is generated by the server or specified by client.

Reply:
Done.